meetings/2019-02-14.md

```diff
@@ -0,0 +1,61 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-02-13
+
+## Links
+
+* **Recording**: https://youtu.be/b8FeRjXC67g
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/664
+* **Minutes Google Doc**: https://docs.google.com/document/d/1ow99XN_btUNt-z-aTa7NCPbB4jKM3g8HCHGREAsJ1xk
+
+## Present
+
+* Colin Ihrig @cjihrig (TSC)
+* Gabriel Schulhof @gabrielschulhof (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Matteo Collina @mcollina (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+
+## Agenda
+
+### Announcements
+ 
+No announcments
+
+
+Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* process: improve nextTick performance [#25461](https://github.com/nodejs/node/pull/25461)
+
+Matteo proposed https://github.com/nodejs/node/pull/25461#discussion_r256348109 to resolve the @apapirovski objection. No one objects.
+
+### nodejs/TSC
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+
+Matteo: Some updates from myles recently to the bootstrap wg.
+Matteo: People should check this stuff out if they are interested in the governance of the new foundation, it is important.
+
+See https://github.com/nodejs/bootstrap/tree/master/proposals
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+
+  - Modules: Our understanding is that the module team is working forward on a minimum implementation 
+  - N-API: Lots of activity with [node-addon-api](https://www.npmjs.com/package/node-addon-api). Working on porting important npm modules. Still in the process of marking n-api as non-experimental.
```

Comment:
I think that would be tsfn as non-experimental as opposed to N-API as N-API is already non-exprimental.